etna/core/models/mixins.py

```diff
@@ -208,3 +214,158 @@ class Meta:
     api_fields = [
         APIField("page_sidebar"),
     ]
+
+
+class SocialMixin(models.Model):
+    """Mixin to add social media sharing options to a Page."""
+
+    teaser_text = models.TextField(
+        verbose_name=_("teaser text"),
+        help_text=_(
+            "A short, enticing description of this page. This will appear in promos and under thumbnails around the site."
+        ),
+        max_length=160,
+    )
+    teaser_image = models.ForeignKey(
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+        help_text=_("Image that will appear on thumbnails and promos around the site."),
+    )
+
+    search_image = models.ForeignKey(

```

Comment:
Could this be `og_image` to fit in with the other properties?

Reply:
I'd like to keep it as `search_image` if that's okay - it was originally this from the `MetadataMixin` from `wagtailmetadata` so keeping it the same will just bring across the images that are already set in there